library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -207,6 +206,11 @@ public void onGlobalLayout() {
             setVisibility(currentLocationBtn, isCurrentLocationBtnVisible);
         }
         featureMap = new HashMap<>();
+
+        // update my location button drawable for mobile
+        if (getResources().getBoolean(R.bool.isTablet)){
+            currentLocationBtn.setImageResource(R.drawable.ic_cross_hair_tablet);
```

Comment:
Is it possible to use the drawables folder for this instead of having this in code?

Is it possible to use drawables configurable for different screen sizes eg. `drawable-sw600dp` and so forth for each mdpi instead of having this in code since that means that you have not covered other DPIs? 

I think it's even easier to manage than having this in code. Yes? No?

Reply:
Fixed.